packages/playwright-test/package.json

```diff
@@ -33,6 +33,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@types/node": "*",
+    "azure-devops-node-api": "^11.2.0",
```

Comment:
Would be amazing if we can get rid of that dependency, this would mean to write our own API client which calls to the azure devops API.

Reply:
it's a great idea but i think it's "out of budget"))

Reply:
Was checking the bundle size, seems definitely a lot, putting it maybe under an extra package, aka `@playwright/ado-reporter` might make sense.

```
npx esbuild --bundle test.js --outfile=out.js --platform=node

out.js  1.7mb ⚠️
```

Reply:
What about making it an optional peer dependency that people manually need to add to the project when using this reporter?